tests/pages/fiche-jeune/Actions.test.tsx

```diff
@@ -468,7 +471,65 @@ describe('Actions dans la fiche jeune', () => {
         // Then
         expect(getActionsJeuneClientSide).toHaveBeenCalledWith('jeune-1', {
           page: 2,
-          statuts: [StatutAction.AFaire],
+          filtres: { statuts: [StatutAction.AFaire], categories: [] },
+          tri: 'date_echeance_decroissante',
+        })
+      })
+    })
+
+    describe('filtrer les actions par catégories', () => {
+      let pageCourante: number
+      beforeEach(async () => {
+        // Given
+        ;(getActionsJeuneClientSide as jest.Mock).mockImplementation(
+          async () => ({
+            actions: [uneAction({ content: 'Action filtrée' })],
+            metadonnees: { nombreTotal: 52, nombrePages: 3 },
+          })
+        )
+        pageCourante = 1
+
+        await renderFicheJeuneMILO({
+          structure: StructureConseiller.MILO,
+          actionsInitiales: {
+            actions,
+            page: pageCourante,
+            metadonnees: { nombreTotal: 52, nombrePages: 6 },
+          },
+          onglet: Onglet.ACTIONS,
+        })
+
+        // When
+        await userEvent.click(screen.getByText('Catégorie'))
+        await userEvent.click(screen.getByLabelText('SNP 1'))
+        await userEvent.click(screen.getByRole('button', { name: 'Valider' }))
+      })
+
+      it('filtre les actions', () => {
```

Comment:
```suggestion
      it('filtre les actions par catégories', () => {
```
pour préciser le type de filtre appliqué

Reply:
idem ligne 450 pour les statuts

Reply:
C'est déjà dans un `describe('filtrer les actions par catégories'`